examples/angular-router/package.json

```diff
@@ -8,30 +8,31 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "6.0.3",
-    "@angular/common": "6.0.3",
-    "@angular/compiler": "6.0.3",
-    "@angular/core": "6.0.3",
-    "@angular/forms": "6.0.3",
-    "@angular/http": "6.0.3",
-    "@angular/platform-browser": "6.0.3",
-    "@angular/platform-browser-dynamic": "6.0.3",
-    "@angular/router": "6.0.3",
-    "angular-instantsearch": "2.0.4",
+    "@angular/animations": "7.0.1",
+    "@angular/common": "7.0.1",
+    "@angular/compiler": "7.0.1",
+    "@angular/core": "7.0.1",
+    "@angular/forms": "7.0.1",
+    "@angular/http": "7.0.1",
+    "@angular/platform-browser": "7.0.1",
+    "@angular/platform-browser-dynamic": "7.0.1",
+    "@angular/router": "7.0.1",
+    "angular-instantsearch": "2.1.0",
     "core-js": "2.4.1",
     "instantsearch.js": "2.3.1",
     "nouislider": "10.1.0",
-    "rxjs": "6.2.0",
+    "rxjs": "6.3.3",
     "zone.js": "0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.6.3",
-    "@angular/cli": "6.0.3",
-    "@angular/compiler-cli": "^6.0.3",
-    "@angular/language-service": "^6.0.3",
+    "@angular-devkit/build-angular": "~0.10.3",
+    "@angular/cli": "7.0.3",
+    "@angular/compiler-cli": "^7.0.1",
+    "@angular/language-service": "^7.0.1",
     "@types/node": "~6.0.60",
-    "codelyzer": "~3.2.0",
+    "codelyzer": "~4.5.0",
     "ts-node": "~3.2.0",
-    "typescript": ">=2.7.0 <2.8.0"
+    "tslint": "~5.0.0",
+    "typescript": ">=2.7.0 <4.0.0"
   }
 }
```

Comment:
Brief overview of what's been updated

- all @angular/* dependencies are updated as part of `ng update` command
- codelyzer was updated to support angular 7 (cf [peer dependencies](https://github.com/mgechev/codelyzer/blob/master/package.json#L98-L99))
- tslint is a new [peerDependency of codelyzer](https://github.com/mgechev/codelyzer/blob/master/package.json#L100) I had to install it
- typescript range was loosened to allow typescript@3 (updating to typescript 3 is recommanded for angular v7 users). 
- angular-instantsearch was updated as older version broke `ng update` (cf. [issue](https://github.com/angular/angular-cli/issues/12644))